src/features/Product/components/AuthorInfo.tsx

```diff
@@ -1,7 +1,14 @@
 import styles from "./AuthorInfo.module.css";
 import profileIcon from "assets/images/ic_profile.svg";
 
-const AuthorInfo = ({ className, nickname, image, date }) => {
+interface Props {
+  className?: string;
+  nickname: string;
+  image?: string;
+  date: string;
+}
```

Comment:
P2:
가능하면 필수값을 위에 두시는 것을 추천드립니다.

```suggestion
interface Props {
  nickname: string;
  className?: string;
  image?: string;
  date: string;
}
```